docs/src/main/asciidoc/logging.adoc

```diff
@@ -796,16 +797,15 @@ However, the APIs are very similar:
 [[mdc-propagation]]
 ==== MDC propagation
 
-Under the hood, Quarkus provides a specific implementation of the MDC provider handling the reactive context.
-Thus, the MDC data is propagated even when using reactive and asynchronous processing.
+In Quarkus, the MDC provider has a specific implementation for handling the reactive context, ensuring that MDC data is propagated during reactive and asynchronous processing.
 
-Consequently, the MDC data is still available:
+As a result, you can still access the MDC data in various scenarios:
 
-- after async calls (like a REST client returning a Uni)
-- in the code submitted to the `ManagedExecutor` (`@Inject org.eclipse.microprofile.context.ManagedExecutor executor`)
-- in the code executed using `vertx.executeBlocking()`
+* After asynchronous calls, like for example, when a REST client returns a Uni.
+* When dealing with code submitted to the `ManagedExecutor` using `@Inject org.eclipse.microprofile.context.ManagedExecutor executor`.
+* In code executed with `vertx.executeBlocking()`.
 
-NOTE: When available, the MDC data is stored on a _duplicated context_ which is an isolated context for your processing.
+NOTE: NOTE: MDC data is stored in a duplicated context, isolated for processing.
```

Comment:
```suggestion
NOTE: If possible, MDC data are stored in a _duplicated context_, which is an isolated context for processing a single task (request).
```

Reply:
In my book, `data` is a plural, but what do I know :-)

Reply:
Thats another bad influence of playing Diablo 4 with all these typos and misconceptions :)